frontend/src/utils/index.js

```diff
@@ -281,3 +273,17 @@ export function deepUnfreeze(obj) {
 export function leftTrim(str) {
   return str.replace(/^\s+/, '');
 }
+
+/**
+ * @method formatSortingQuery
+ * @summary format's the ordering parameters prefixing them with asc/desc sign
+ */
+export function formatSortingQuery(orderBy) {
+  if (orderBy && orderBy.reduce) {
```

Comment:
is there any reason to check for `reduce` instead of `map` which is used 1 line below?

Reply:
My previous version used reduce here, but it doesn't really matter. Can be any of them as they're both `array`-specific